axelrod/tests/strategies/test_graaskamp.py

```diff
@@ -0,0 +1,196 @@
+"""Tests for the Graaskamp strategies."""
+
+import axelrod
+from .test_player import TestPlayer
+
+C, D = axelrod.Action.C, axelrod.Action.D
+
+class TestMoreGraaskamp(TestPlayer):
```

Comment:
These tests look good but can you add an `attrs` check (for each) which verifies the strategy is in the mode it should be. An example here: https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/strategies/test_oncebitten.py#L38